splice_machine/src/main/java/com/splicemachine/derby/impl/load/HdfsImport.java

```diff
@@ -770,6 +828,9 @@ private static void doImport(String schemaName,
                 rs.open();
                 results[0] = new EmbedResultSet40((EmbedConnection) conn, rs, false, null, true);
             } catch (SQLException | StandardException | IOException e) {
+                if( isLoadReplaceMode ) {
+                    conn.rollback();
```

Comment:
makes sense to check, that conn != null, as SpliceAdmin.getDefaultConn can throw SQLException

Reply:
good find! fixed